src/dnstest_harness.erl

```diff
@@ -11,7 +11,7 @@
 -export([init/1, handle_call/3, handle_cast/2]).
 
 % Internal API
--export([send_request/3]).
+-export([send_request/3, send_request/4]).
```

Comment:
Any reason to keep the one with arity 3 available? If we don't need it, we might remove it.

Reply:
Changed in [39697b8](https://github.com/dnsimple/dnstest/pull/21/commits/39697b81b7bd2f8ab501e98e0bf63c120dee3448)